src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.cpp

```diff
@@ -100,7 +100,8 @@ ClosureRemap build_remap(const Function& fbody, const DCOFFParams& params_to) {
         auto zerop_iter = params_to.zerops.find(param);
         if (zerop_iter != params_to.zerops.end()) {
             LOG_DEBUG("This parameter requires zero point: " << zerop_iter->second);
-            m.zero_points.push_back(ov::npuw::util::tensor_from_const(zerop_iter->second));
+            m.zero_points.push_back(
+                ov::npuw::util::tensor_from_const(std::dynamic_pointer_cast<ov::op::v0::Constant>(zerop_iter->second)));
```

Comment:
what was the reason to change this?

Reply:
To support DCOFF I believe

Reply:
you can just revert it back as I mentioned. THere's no need for change like this.